.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,20 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/javascript-node
+{
+	"name": "Node.js",
+	"image": "mcr.microsoft.com/devcontainers/javascript-node:0-16-bullseye",
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	"features": {
+		"ghcr.io/carlosdp/devcontainer-features/foundry:latest": {}
+	},
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	"forwardPorts": [
+		3000
+	],
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"postCreateCommand": "yarn install && yarn prepare"
```

Comment:
`prepare` should happen automatically post-install, so I don't think this is needed?

Can also reduce this to just `yarn`

Reply:
It doesn't run automatically on Yarn v2+ https://github.com/yarnpkg/berry/issues/2305#issuecomment-753468942

Reply:
TIL!